app/assets/javascripts/disqus.js

```diff
@@ -0,0 +1,14 @@
()}},preventScrolling:function(){var a=this.getBodyOverflow(),b=document.body.style.marginRight,d=document.documentElement.style,e=d.overflow;this.listenToOnce(this,"frame:close",function(){this.setBodyStyles({overflow:a,marginRight:b}),d.overflow=e}),this.setBodyStyles({overflow:"hidden",marginRight:this.calcMargin(c.getComputedStyle(document.body,"margin-right","marginRight")||b)}),d.overflow="hidden"},triggerHostReflow:function(){var a=document.createElement("style");document.body.appendChild(a),document.body.removeChild(a)},calcMargin:function(a){var b=a.match(e);return b&&(a=b[1]),a?"calc("+a+" + "+c.getScrollbarWidth()+"px)":c.getScrollbarWidth()+"px"},setBodyStyles:function(a){for(var b in a)document.body.style[b]=a[b]},getBodyOverflow:function(){return document.body.style.overflow},getSecureOrigin:function(){var a=b.urls.ensureHttpBasedProtocol("https://disqus.com/home/","https:"),c=a.split("/"),d=c[0],e=c[2];return d+"//"+e},getFrameSettings:function(a){return a.role="dialog",a.origin=this.getSecureOrigin(),a.styles={height:"100%",position:"fixed",top:0,right:0,left:"auto",bottom:"auto","z-index":c.MAX_Z_INDEX,visibility:"hidden"},a},getUrl:function(){var a=this.settings.path||"",c=this.settings.language,d={utm_source:"disqus_embed"};return c&&"en"!==c&&(d.l=c),DISQUS.serialize(b.urls.apps[this.name]+a,d)},show:function(a){if(!this.frame.isReady())return void this.once("frame:ready",function(){this.show(a)},this);var b={path:a};this.settings.sso&&(b.sso=this.settings.sso),this.frame.sendMessage("showPath",b)}},{getInstanceOrLoad:function(a){var b=f.instance;return b?b:(b=f.instance=new f(a),a.preload&&b.listenToOnce(b,"afterInit",function(){b.frame.hide()}),b.init(),b)},preload:function(a){return a.preload=!0,f.getInstanceOrLoad(a)},destroy:function(){var a=f.instance;a&&(a.destroy(),f.instance=null)},show:function(a){var b=f.getInstanceOrLoad(a);return b.show(a.path),b}});return{show:f.show,preload:f.preload,destroy:f.destroy,_HomeApp:f}}),DISQUS.define("next.host.lounge",function(a){"use strict";var b=a.document,c=".disqus-loader{animation:disqus-embed-spinner .7s infinite linear;-webkit-animation:disqus-embed-spinner .7s infinite linear}@keyframes disqus-embed-spinner{0%{transform:rotate(0deg)}100%{transform:rotate(360deg)}}@-webkit-keyframes disqus-embed-spinner{0%{-webkit-transform:rotate(0deg)}100%{-webkit-transform:rotate(360deg)}}",d="//a.disquscdn.com/next/assets/img/loader-bg.a99c37cfdf7091ca7465f065716ef653.png",e=DISQUS.next.host,f=e.utils,g=e.app.ThreadBoundApp,h=g.extend({name:"lounge",loaderVersion:f.getLoaderVersionFromUrl("//a.disquscdn.com/next/lounge.load.bbf5c12736401f49654a1fa147dc558d.js"),indicators:null,wasInViewport:!1,triggeredSlowEvent:!1,events:{beforeInit:function(){var a=this.settings;a.unsupported||(this.indicators={},this.isContainerVisible()?this.addLoadingAnim():this.addLoadingAnimOnContainerVisible(),this.bindPublisherCallbacks(),this.forwardGlobalEvents())},"frame:reload":function(){a.location.reload()
```

Comment:
Line is too long.<br>Missing semicolon.<br>Expected '{' and instead saw 'for'.<br>Expected '{' and instead saw 'c'.<br>Expected an assignment or function call and instead saw an expression.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.<br>Expected '{' and instead saw 'e'.<br>Expected '{' and instead saw 'b'.<br>Expected '{' and instead saw 'a'.<br>Missing '()' invoking a constructor.<br>Too many errors. (73% scanned).